lib/core/src/input/input.tsx

```diff
@@ -56,61 +59,68 @@ export interface InputProps {
 	// Style
 
 	/**
-	 * Icon appear in an input. See the [Icons guide][1] to learn more.
+	 * Icon in the Input. See the [Icons guide][1] to learn more.
```

Comment:
(3) But this one have

Reply:
Following https://github.com/moaijs/moai/pull/201#discussion_r640286270, this one does have full stops since there are 2 of them, so we need something to separate them, which is more important than the above rules. Note that "Icon in the Input" is still not a complete sentence (when it stands alone, you don't know what is its subject and verb). The full stops here are only to separate them. The last full stop may not needed technically, but I put it there to keep it consistent.